File: Python/keyboard-row.py
class Solution:
    def findWords(self, words):
        """
        :type words: List[str]
        :rtype: List[str]
        """
        res = []
        rows = ['qwertyuiop', 'asdfghjkl', 'zxcvbnm']

        for word in words:
            for row in rows:
                if all(letter in row for letter in word.lower()):
                    res.append(word)

        return res

File: Swift/keyboard-row.swift
class Solution {
    func findWords(_ words: [String]) -> [String] {
        let rowOne = "qwertyuiop"
        let rowTwo = "asdfghjkl"
        let rowThree = "zxcvbnm"

        return words.filter { contain(rowOne, $0) || contain(rowTwo, $0) || contain(rowThree, $0) }
    }

    private func contain(_ str: String, _ word: String) -> Bool {
        return word.lowercased().filter { !str.contains($0) }.count == 0
    }
}
